### ISSUE TITLE
Program stucks and training is not going on

### ISSUE THREAD

**Issue by PaulZhangIsing** [shaohua0116/WGAN-GP-TensorFlow#5]
I tried to download the whole file, and downloaded CIFAR100 dataset.
while I try to run the script:
 python3 trainer.py --dataset CIFAR100  --batch_size 25 --num_dis_conv 6 --gan_type wgan-gp
I waited for one day and longer, however, it seems it stucks there and does not progress.

{'id': <tf.Tensor 'shuffle_batch:0' shape=(25,) dtype=string>,
 'image': <tf.Tensor 'shuffle_batch:1' shape=(25, 224, 224, 3) dtype=float32>}
2019-10-18 16:18:03.281299: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcublas.so.9.0 locally
0 10000000 <tf.Variable 'global_step:0' shape=() dtype=int64_ref>
'**************now is at step 0*********************'
[2019-10-18 16:18:04,956] Saved checkpoint at 0
